### DISCUSSION TITLE
Bug with context_menus example.

### DISCUSSION THREAD

**Discussion by btassone** [bwmarrin/discordgo#1212] [Q&A] [answered]
Hey was playin around with the context_menus example and was trying to compile it but running into:

"cannot use discordgo.MessageFlagsEphemeral (constant 64 of type discordgo.MessageFlags) as type uint64 in struct literal"

Is there something i'm missing thats simple to fix this?

**Comment by btassone** [comment]
Sorry the fix was to just cast it to uint64.

**Comment by FedorLap2006** [comment, marked as answer]
I suspect you're using `@latest` version. However example uses `@master` (i.e. code from `master` branch version).
There was a change in types recently, that's why you're getting this error.

The current fix is, yes, conversion to `uint64`, but once `v0.25.0` comes out, you will no longer need it.
